fmt/src/formatter.rs

```diff
@@ -377,6 +377,9 @@ macro_rules! writeln_chunk {
 }
 
 macro_rules! write_chunk_spaced {
+    ($self:expr, $loc:expr, $needs_space:expr, $format_str:literal) => {{
+        write_chunk_spaced!($self, $loc, $needs_space, $format_str,)
+    }};
```

Comment:
as a followup we could move all macros to a separate `macros` module

Reply:
sounds good, will include in the next PR